src/fast-path.js

```diff
@@ -380,7 +380,11 @@ FPp.needsParens = function(assumeExpressionContext) {
       }
 
     case "ClassExpression":
-      return parent.type === "ExpressionStatement";
+      switch (parent.type) {
+        case "ExportDefaultDeclaration":
+        case "ExpressionStatement":
+          return true;
+      }
```

Comment:
After the switch, I mean, so that it doesn't fall through to ObjectExpression. Unless you meant it to?